### DISCUSSION TITLE
💡 Output telegrams to a network port

### DISCUSSION THREAD

**Discussion by Calimerorulez** [dsmrreader/dsmr-reader#1509] [Questions & answers / How-to's] [answered]
### Feature

Hi,

As I'm running dsmr-reader in a dockerized container, with exclusive access to the USB port on which the P1 adapter is connected, it would be nice if dsmr-reader could 'netcat' (or 'socat' or something like that) the received telegrams to a network port, so I can connect other apps to that port.

I know the existence of the API and MQTT, and I use it too, but above feature would be nice (for me :) )

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je verzoek. Ik denk dat je het beste gebruik kan maken van `ser2net`. 

Sinds [DSMR-reader 4.1 kun je namelijk een netwerkpoort laten uitlezen](https://github.com/dsmrreader/dsmr-reader/issues/1057#issuecomment-665297541). Zie ook bijvoorbeeld [dit issue over hoe je het kan instellen en ook dat die meerdere connecties aan kan](https://github.com/dsmrreader/dsmr-reader/issues/645#issuecomment-496497500).

Het is dan aan jou om te bepalen waar je `ser2net` draait, maar dat kan in principe overal gemakkelijk op een Linux OS. Of op je Docker host of in een (andere) docker container. Daar heeft iemand vast wel wat voor gemaakt.

En dan kun je zowel DSMR-reader als je andere applicaties er aan hangen.
